From 6f5a4ad33580a719397bd58eeecbef065910c65a Mon Sep 17 00:00:00 2001
From: Arnout Engelen <arnouten@bzzt.net>
Date: Thu, 26 Apr 2012 13:57:57 +0200
Subject: [PATCH] add Homepage field, pointing to the Debbugs Team page (as
 that seems the most useful page for users of these
 packages, rather than the end-user-oriented
 http://bugs.debian.org)

---
 debian/control |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/debian/control b/debian/control
index 732f98c..7ed0d89 100644
--- a/debian/control
+++ b/debian/control
@@ -13,6 +13,7 @@ Build-Depends-Indep: libparams-validate-perl,
  libmail-rfc822-address-perl, libsafe-hole-perl, libuser-perl,
  libconfig-simple-perl, libtest-pod-perl, liblist-moreutils-perl,
  libtext-template-perl
+Homepage: http://wiki.debian.org/Teams/Debbugs
 
 Package: debbugs
 Architecture: all
-- 
1.7.9.1

